wiki/wikiproject_klara_weißbender.qmd

```diff
@@ -0,0 +1,21 @@
+
+Relationship: A lot of studies use the connectivity paradox from Vanden Abeele (2021) as a baseline for their empirical work. A lot of these studies also derive variables from this paper and essentially say the same thing as the study of Gilbert et al.. Most of the time these papers are about psychological well being and perceived autonomy just like Gilbert et al.. For example the study of Granow et al. (2018) deals with these themes as well. But it just chooses a different starting point and deals with the phenomenen of binge watching. 
+
+Questions: Further research should explore whether online vigilance and autonomy need dissatisfaction are the only two situational boundary conditions or whether there are more. And it should be explored if other situational boundary conditions show bigger interaction effects with the two proposed predictors of stress on the within-person level.Because online vigilance and autonomy need dissatisfaction only show small ones. 
+
+
+Summary: The study of Gilbert et al. (2022) focuses on the mobile connectivity and its negative effect on smartphone users by causing stress. The study tries to throw a new perspective on the research field of digital stress by using the mobile connectivity paradox by Vanden Abeele and the IM3UNE model by Schneider et al. which both assume that digital stress effects could be conditional. Gilbert et al. introduces two situational boundary conditions -goal conflict and autonomy need dissatisfaction- in order to research this assumption.The assumption is split into seven different hypotheses in order to be measurable.The seven different hypotheses focus on the investigation of the relationships between the cognitive-motivational (online vigilance) and the behavioral (media multitasking and communication load) smartphone use patterns. In addition the connection between these use patterns and perceived stress is examined against the background of the situational boundary conditions (goal conflict and autonomy need dissatisfaction). The hypotheses are tested based on data from a 7-day experience sampling study. Moreover the whole study design is supposed to address the within-person dynamics of digital stress. The results show that online vigilance can cause stress directly and does so especially through the increase of the communication load. The two situational boundary conditions (goal conflict and autonomy need dissatisfaction) moderate the influence of online vigilance and media multitasking on stress. To conclude the study extends the focus of the between-person associations to a focus on the within-person fluctuations in the context of digital stress. The study also implicates that it's really important to reduce digital stress. Given that online vigilance and communication load cause stress, it seems relevant to establish a conscious management of flows of online communication and make it possible to reduce stress at work and in private. 
```

Comment:
Might other perspectives on autonomy need experiences have different effects here? This study focuses on media use related autonomy need frustration — how about other levels (think for example about the spheres of experience discussed in a prior session).

Reply:
what does the Im3une Model exactly say in the context of autonomy?

Reply:
The IM3UNE assumes that digital stress effects could be conditional. And you can look up the exact model in the third presentation of this class, because we already had that.